tests/Common/CLITestCase.php

```diff
@@ -1,6 +1,6 @@
 <?php
```

Comment:
This was moved from `tests/Integrations` to `test/Common` in order to be properly used all over the test suites.